src/wristpy/core/orchestrator.py

```diff
@@ -254,7 +268,7 @@ def _run_file(
             either .csv or .parquet formats.
         thresholds: The cut points for the light, moderate, and vigorous thresholds,
             given in that order. Values must be ascending, unique, and greater than 0.
-            Default values are optimized for subjects ages 7-11 [1].
+            Default values are optimized for subjects ages 7-11 [1][3].
```

Comment:
```suggestion
            Default values are optimized for subjects ages 7-11 [1] - [3].
```
I forgot to add [2] previously.